frontend/src/admin/components/sidebar/dropdown/dropdownMenu.tsx

```diff
@@ -0,0 +1,33 @@
+import React, {useState} from "react";
+import MenuIcon from "./menuIcon";
+import Dropdown from "./dropdown";
+import "./dropdownMenu.scss";
+
+export default function DropdownMenu() {
```

Comment:
Refer to my other comment on `DropDown` vs `Dropdown`

Reply:
✅